src/about/IntroContainer.css

```diff
@@ -22,15 +21,35 @@
 
 .intro {
   width: 60%;
+  text-align: center;
```

Comment:
Also, for wherever this text does go, I don't think we should center it. Website has been leaning towards left and right alignment